internal/core/asset/asset_controller.go

```diff
@@ -288,3 +300,35 @@ func buildSBOM(asset models.Asset, version string, organizationName string, comp
 	bom.Components = &bomComponents
 	return &bom
 }
+
+func (c *httpController) UpdateRequirements(ctx core.Context) error {
+	asset := core.GetAsset(ctx)
+
+	req := ctx.Request().Body
+	defer req.Close()
+
+	var assetNew models.Asset
+	err := json.NewDecoder(req).Decode(&assetNew)
+	if err != nil {
+		return fmt.Errorf("Error decoding request: %v", err)
+	}
+
+	if assetNew.ConfidentialityRequirement != asset.ConfidentialityRequirement || assetNew.IntegrityRequirement != asset.IntegrityRequirement || assetNew.AvailabilityRequirement != asset.AvailabilityRequirement {
+		justification := "Requirements Level updated: " + "AvailabilityRequirement: " + asset.AvailabilityRequirement + " -> " + assetNew.AvailabilityRequirement + ", ConfidentialityRequirement: " + asset.ConfidentialityRequirement + " -> " + assetNew.ConfidentialityRequirement + ", IntegrityRequirement: " + asset.IntegrityRequirement + " -> " + assetNew.IntegrityRequirement
+		justificationStr := string(justification)
+
+		asset.ConfidentialityRequirement = assetNew.ConfidentialityRequirement
+		asset.IntegrityRequirement = assetNew.IntegrityRequirement
+		asset.AvailabilityRequirement = assetNew.AvailabilityRequirement
+
+		err = c.assetService.UpdateAssetRequirements(asset, core.GetSession(ctx).GetUserID(), justificationStr)
+		if err != nil {
+			return fmt.Errorf("Error updating requirements: %v", err)
+		}
+
+		return nil
```

Comment:
Currently you are not returning any content at all after the update happens. It would be great if you would just return the updated asset as json response. What do you think?

```suggestion
		return ctx.JSON(200, asset)
```